AppKit/CPWindow/CPWindow.j

```diff
@@ -198,6 +199,8 @@ var CPWindowActionMessageKeys = [
     CPView                              _toolbarView;
 
     CPArray                             _mouseEnteredStack;
+    CPArray                             mouseEnteredStack;
```

Comment:
Indeed it was no more useful so I removed it. This is some relic of my proposed fix for multiple mouse events fires.